conf/views/macros/title.njk

```diff
@@ -1,5 +1,5 @@
 {% macro title(mainTitle, errors = []) %}
-  {% set titlePrefix = messages("error.title.prefix") if errors.length > 0 else "" %}
+  {% set titlePrefix = messages("error.title.prefix") + " " if errors.length > 0 else "" %}
 
-  {{ titlePrefix + ' ' + mainTitle + ' ' + messages("site.title.suffix") }}
+  {{ titlePrefix + mainTitle + ' - ' + messages("site.service_name") + ' ' + messages("site.title.suffix") }}
```

Comment:
Is the space between `titlePrefix` and `mainTitle` no longer necessary?

Reply:
I've moved the space to the line above where titlePrefix is set to avoid a preceding space when there is no prefix